data/channels.csv

```diff
@@ -29067,4 +29074,24 @@ Vivamovil.es,Vivamóvil,,,ES,ES-AN,,s/ES-AN,spa,,FALSE,,,,https://vivamovil.tv/e
\ No newline at end of file
+LaFabricaTV.es,La Fabrica TV,,,ES,ES-AN,,c/ES,spa,,FALSE,,,,https://app.lafabricatv.es/canal-en-directo,https://app.lafabricatv.es/apiv3/release/iphone/10/photo/sections_38254038_navBar_titleImage@2x.png
+BiosferaTV.es,Biosfera TV,,,ES,ES-CN,,s/ES-CN,spa,,FALSE,,,,https://www.biosferadigital.com/estaticas/biosferatvdirecto/,https://i.imgur.com/xv4AM5I.png
+Nortevision.es,Nortevisión,,,ES,ES-CN,,s/ES-CN,spa,,FALSE,,,,http://nortevision.es/directo/,https://i.imgur.com/VGOAGMK.png
+AfortunadasTV.es,Afortunadas TV,,,ES,ES-CN,Arucas,s/ES-CN,spa,,FALSE,,,,https://afortunadastv.com/,https://i.imgur.com/TcEoUPA.png
+TVHellin.es,TV Hellin,,,ES,ES-CM,Hellin,s/ES-CM,spa,,FALSE,,,,https://televisionhellin.com/tv-en-directo/,https://televisionhellin.com/wp-content/uploads/2021/08/logotipo-tvhellinonline.png
+TVBenavente.es,TV Benavente,,,ES,ES-CM,Benavente,s/ES-CM,spa,,FALSE,,,,https://www.tvbenavente.es/directo/,https://i.imgur.com/5dSHBcZ.png
+BDN.es,BDN,,,ES,ES-CT,Badalona,s/ES-CT,cat,,FALSE,,,,https://www.bdncom.cat/,https://www.bdncom.cat/img/logoTeleb.png
+Tac12.es,Tac12,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://www.tac12.tv/directe/index.html,https://www.tac12.tv/directe/img/TAC12_tv_logo.png
+CanalTerrassa.es,Canal Terrassa,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://terrassadigital.cat/televisio/,https://i.imgur.com/NkflkLi.png
+TVCostaBrava.es,TV Costa Brava,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://www.tvcostabrava.com/canals-online/televisio-costa-brava-online,https://www.tvcostabrava.com/images/logo.png
+OlotTV.es,Olot TV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,https://www.olot.tv/olot-televisio-en-directe/,https://i.imgur.com/rXMWUvL.png
+TVLHospitalet.es,TV L'Hospitalet,,,ES,ES-CT,L'Hospitalet,s/ES-CT,cat,,FALSE,,,,https://lhdigital.cat/web/digital-h/en-directe,https://upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Logotip_TV_L%27H.svg/400px-Logotip_TV_L%27H.svg.png
+TVBergueda.es,TV Berguedà,,,ES,ES-CT,Bergueda,s/ES-CT,cat,,FALSE,,,,https://tvbergueda.alacarta.cat/,https://www.alacarta.cat/api/assets/getFile/public/media/000/793/324/logoalfa.png
+RTVElVendrell.es,RTV El Vendrell,,,ES,ES-CT,El Vendrell,s/ES-CT,cat,,FALSE,,,,https://www.rtvelvendrell.cat/directes/televisio/,https://www.rtvelvendrell.cat/wp-content/uploads/2020/09/logo-rtv-reduit-1.png
+VOTV.es,VOTV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://votv.alacarta.cat/votv-en-directe,https://i.imgur.com/1g2Z64L.png
+PieraTV.es,Piera TV,,,ES,ES-CT,,s/ES-CT,cat,,FALSE,,,,http://08-produccions.cat/pieratv/,https://yt3.ggpht.com/WtsIUXtvmo0QQFYhj9qbxpttucZF0ZElx8oju9r7ISrKfn9GHBuRR7s9JN4YRnzvbLr6GiUtWQ=s200-c-k-c0x00ffffff-no-rj
+RTVCCardedeu.es,RTVC,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png

```

Comment:
```suggestion
RTVC.es,RTVC,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png
```

Reply:
There are two channels similar to RTVC. One is in the Canarias and the other in Catalonia (see the ".es" and ".cat" from the official website).

Reply:
Probably best to change the name then, so there is no confusion.
```suggestion
RTVCCardedeu.es,RTVC Cardedeu,,,ES,ES-CT,Cardedeu,s/ES-CT,cat,,FALSE,,,,https://www.rtvc.cat/tele/,https://www.rtvc.cat/wp-content/uploads/2020/06/LOGO_RTVC_WEB.png
```

Reply:
Done. Thank you for this idea.